core/src/main/java/jenkins/management/EditDescriptionLink.java

```diff
@@ -0,0 +1,28 @@
+package jenkins.management;
+
+import hudson.Extension;
+import hudson.model.ManagementLink;
+import org.jenkinsci.Symbol;
+
+@Extension(ordinal = Integer.MAX_VALUE - 800) @Symbol("editDescription")
```

Comment:
Ideally you shouldn't play with ordinals. They are used for Descriptor ordering and because almost all jenkins iterating on this Descriptor lists for contributing into page pieces of UI it influences on UI order.